downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -73,6 +73,10 @@ To view bug fixes, see the [GitHub milestone for 2201.4.0 (Swan Lake)](https://g
 - Added support for multiplexing in GraphQL subscriptions
 - Added support to access the GraphQL field information from resolvers
 
+##### `nats` package
+
+- Added support for NATS JetStream client, publishing and subscribing
```

Comment:
```suggestion
- Added support for publishing and subscribing of the NATS JetStream client
```

Reply:
This seems not right. Maybe this?

```suggestion
- Added support for NATS JetStream client with publishing and subscribing
```

Reply:
Shall we say, `Added support for the NATS JetStream client with publishing and subscribing functionalities`

Reply:
Does it convey that `JetStream client` itself is entirely new

Reply:
How about saying, `Added support for the new NATS JetStream client with publishing and subscribing functionalities`

Reply:
fixed please approve